src/format/kdbx4/mod.rs

```diff
@@ -66,11 +61,12 @@ mod kdbx4_tests {
         inner_cipher_suite: InnerCipherSuite,
         kdf_setting: KdfSettings,
     ) {
-        let mut db = Database::new(NewDatabaseSettings {
+        let mut db = Database::new(DatabaseSettings {
+            version: DatabaseVersion::KDB4(0),
```

Comment:
@sseemayer we have a constant value for the inner version number, called `KDBX4_CURRENT_MINOR_VERSION`

Reply:
Right! I did use this in another location but must have missed this one. #121 should fix it.